dotcom-rendering/src/components/Card/components/LI.tsx

```diff
@@ -20,10 +20,13 @@ const liStyles = css`
 
 const sidePaddingStyles = (padSidesOnMobile: boolean) => css`
 	/* Set spacing on the li element */
-	${padSidesOnMobile && until.tablet} {
-		padding-left: 10px;
-		padding-right: 10px;
-	}
+	${padSidesOnMobile &&
+	css`
```

Comment:
I'd be tempted to just make this a separate style from `sidePaddingStyles` and make `sidePaddingStyles` a constant. Then apply the new style conditionally on https://github.com/guardian/dotcom-rendering/pull/7892/files#diff-9299959c0daa560a3f8c2bea4463987f988f7ec9de1332ac40a1954f653fdd73L116

From an Emotion standpoint I think this is _very slightly_ better as Emotion only has to serialize the CSS once if its a constant not a function!